shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -721,7 +721,8 @@ \subsection{Block Header Transition}
         \vdash \var{ru}\trans{\hyperref[fig:rules:reward-update]{rupd}}{\var{s}} \var{ru}'
       }
       \\
-      \var{nes}''\leteq\var{nes'}\unionoverrideRight\{\var{ru'}\}
+      \var{nes}''\leteq\var{nes'}\unionoverrideRight\{\var{ru'}\} % TODO: should
+                                % that simply replace the reward update in nes'?
```

Comment:
yes, it was an abuse of notation in the spec. I can change the spec if you think it is confusing.